packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div :class="[$style['c-restaurantCard-fees']]">
+        <icon-text
+            v-if="deliveryFeeText"
+            :class="[$style['c-restaurantCard-fees-item']]"
+            :text="deliveryFeeText"
+            :hide-icon-in-tile-view="true"
+            data-test-id="restaurant-fees-delivery">
+            <cash-small-icon data-test-id="restaurant-fees-delivery-icon" />
+        </icon-text>
+
+        <icon-text
+            v-if="minOrderText"
+            :class="[$style['c-restaurantCard-fees-item']]"
+            :text="minOrderText"
+            data-test-id="restaurant-fees-min-order" />
+    </div>
+</template>
+
+<script>
+import { CashSmallIcon } from '@justeat/f-vue-icons';
+import IconText from '../IconText.vue';
+
+export default {
+    name: 'RestaurantFees',
+    components: {
+        IconText,
+        CashSmallIcon
+    },
+    props: {
+        deliveryFeeText: {
+            type: String,
+            default: null
+        },
+        minOrderText: {
+            type: String,
+            default: null
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-restaurantCard-fees {
+    display: flex;
+    align-items: center;
+}
+
+.c-restaurantCard-fees-item {
+    position: relative;
+    padding-right: spacing();
+    margin-right: spacing(x0.5);
+    white-space: nowrap;
+    font-size: $font-body-s-paragraph * 1px;
```

Comment:
You can use the font-size mixin to convert this to relative units:

`@include font-size($font-body-s-paragraph);`

Reply:
cheers!